challenges/objects-arrays.js

```diff
@@ -97,29 +224,30 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
-console.log(lowerCase); 
+const lowerCase = zooAnimals.map(animal => animal["animal_name"].toLowerCase());
+console.log(lowerCase);
 
 /* Request 3: .filter() 
 
-The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
+The zoos are concerned about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+const lowerPopulation = zooAnimals.filter(animal => animal["population"] < 5);
```

Comment:
Excellent work! Advanced array methods and arrow syntax present throughout this whole file 👍